interp/scope.go

```diff
@@ -164,6 +164,8 @@ func (interp *Interpreter) initScopePkg(n *node) (*scope, string) {
 		pkgName = p.child[0].ident
 	}
 
+	interp.mutex.Lock()
+	defer interp.mutex.Unlock()
```

Comment:
This can be a direct call to unlock since there's only a single exit here, and it's close. This would save on defer cost.